pkg/sfu/downtrack.go

```diff
@@ -552,6 +578,13 @@ func (d *DownTrack) Mute(muted bool) {
 	if d.onSubscriptionChanged != nil {
 		d.onSubscriptionChanged(d)
 	}
+
```

Comment:
I think this should go at the top of the function. Otherwise, the blank frames are getting written after the `forwarder.mute` call, which causes them to get sent after the track comes off of mute.

Reply:
I have verified that it works fine @bekriebel . `forwarder` is not doing in terms of actually forwarding packets. It is just calculating forwarding parameters. I logged all the packets getting sent in a debug build locally and it was sending the silence frames on mute. I will double check that just to be sure.

I am wondering if bunching up all the silence frames (although we put the proper sequence number/timestamp) is causing some effect on client side and the client side jitter buffer to throw away packets. Maybe, using a go routine to send them at proper interval is the way to go. I will try to repro first with these changes and see.

Reply:
You're totally right about the position of the call, sorry. I must have just had some added latency in my connection when I ran that test and assumed the silence was causing it at the start of the unmute. I just tested again 3 seconds of silence and it works as expected.

I'm having a hard time getting solid reproduction. I think this helps a lot. I'm still able to occasionally cause the static to occur with this. Testing with the `RTPBlankFramesMax = 10`, I've never been able to get it to reproduce, but it's also entirely possible that is just fluke.

I'd be okay with it being released as-is, however I think it'd be a slight improvement if the "blank frame max" was set to cover 200ms worth of frames the same it is with video. Would it work to have `RTPBlankFramesMax` be a time value in seconds (0.2) and then multiply it by the frame rate to get the proper value?

Reply:
No worries @bekriebel . 

I will try to a goroutine approach to send silence periodically after mute (hoping to find time today to try that). We can try that. If that does not show improvement, I will merge this with number of blank frames bumped up to 10.

Reply:
Sounds good. If you think it is better to do it with the goroutine, I'm all for it. This is what I was thinking as far as basing it on frameRate. So far, I haven't reproduced the issue with a build running with this change: https://github.com/bekriebel/livekit-server/commit/07ae8868b3462e8918df25b8d1a6182d9a32eebf

Reply:
Thank you @bekriebel . Will have a look.

Reply:
I was just able to reproduce the issue once. This time, I a bit of latency to my connection (jumping through a VPN to add an extra hop). I wonder if the problem is increased by latency to the room or a poorer network connection?

This would explain why one of my particular users is having this issue; he's connecting with people across the globe so several of them have higher latency to the room node. I can test the latency theory more tomorrow.

Regardless, I think your change is a good one since it has made it much harder to cause the issue for me :-)

Reply:
Ah, that's very interesting. I can try with more latency.